app/models/wpcc/full_contribution_calculator.rb

```diff
@@ -5,12 +5,17 @@ def eligible_salary
     end
 
     def employee_percent
-      1
+      percentage(:employee)
     end
 
     def employer_percent
-      return 0 if salary_per_year < lower_earnings_threshold
-      return 1 if salary_per_year >= lower_earnings_threshold
+      percentage(:employer)
+    end
+
+    private
+
+    def below_threshold
```

Comment:
Since it is returning a boolean can we rename to below_threshold?

Reply:
Sure